theme_bewise/views/new_page_template.xml

```diff
@@ -51,9 +51,24 @@
     </xpath>
 </template>
 
+<template id="new_page_template_about_full_s_numbers" inherit_id="website.new_page_template_about_full_s_numbers">
+    <xpath expr="//section" position="attributes">
+        <!-- Added by both theme and new page template -->
+        <attribute name="class" add="o_cc3" remove="o_cc3" separator=" "/>
+    </xpath>
+</template>
+
+<template id="new_page_template_about_mini_s_cover" inherit_id="website.new_page_template_about_mini_s_cover">
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" remove="pt40 pb40" separator=" "/>
+    </xpath>
+</template>
+
 <template id="new_page_template_about_personal_s_numbers" inherit_id="website.new_page_template_about_personal_s_numbers">
     <!-- Remove shape option -->
     <xpath expr="//section" position="attributes">
+        <!-- Added by both theme and new page template -->
+        <attribute name="class" add="o_cc3" remove="o_cc3" separator=" "/>
```

Comment:
Remove is useless (or the entire line) -> same value in add and remove
There is 4 other occurences of this

Reply:
This is to avoid a duplicate `o_cc3`: `remove` removes all occurrences, `add` adds only the specified one.
(That's why I added a comment above those situations to mention that the class had been added from several templates)